### PR TITLE
feat: toggle select options on focus

### PR DESCRIPTION
**Pull request by MltStephane**
Small QOL improvement. With this minor change, options are opened when select is focused

### CODE REVIEW

File: resources/views/components/select.blade.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ class="mb-1"
             class="cursor-pointer overflow-hidden !text-transparent !dark:text-transparent"
             x-ref="input"
             x-on:click="toggle"
+            x-on:focus="toggle"
```
Comment (ph7jack) [inline resources/views/components/select.blade.php]: Using the toggle in the focus event will stop opening the select when clicking on it.
The focus event should open the popover.

**Review by ph7jack — CHANGES_REQUESTED** [review]

File: resources/views/components/select.blade.php (hunk 1 above)
Comment (MltStephane) [reply resources/views/components/select.blade.php]: Should I set popover to true on the focus, or remove the focus instruction in the toggle method ?

Comment (ph7jack) [reply resources/views/components/select.blade.php]: Yes, just call the open method.

**Review by ph7jack — APPROVED** [review]

**Comment by ph7jack** [comment]
@MltStephane thanks for contibuting

**Pull request merged**